policyengine_api/routes/policy_routes.py

```diff
@@ -0,0 +1,125 @@
+from flask import Blueprint, Response, request
+import json
+
+from policyengine_api.services.policy_service import PolicyService
+from policyengine_api.utils.payload_validators import (
+    validate_country,
+    validate_set_policy_payload,
+)
+
+policy_bp = Blueprint("policy", __name__)
+policy_service = PolicyService()
+
+
+@policy_bp.route("/<policy_id>", methods=["GET"])
+@validate_country
+def get_policy(country_id: str, policy_id: int | str) -> Response:
+    """
+    Get policy data for a given country and policy ID.
+
+    Args:
+        country_id (str): The
+        policy_id (int | str): The policy ID.
+
+    Returns:
+        Response: A Flask response object containing the
+        policy data in JSON format
+    """
+
+    if policy_id is None:
```

Comment:
**question** Is this even possible? I would expect the path to just not match.

If possible, love to learn how. If not possible, **suggestion** remove the unexecuted code.